### DISCUSSION TITLE
Enable selected Swagger API

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#2114] [General] [answered]
Currently Oqtane provide enable or disable swagger APIs. It would be great if we can enable selected APIs in the system.

**Use case:** We have multiple departments such as Mobility, Web, etc., Each one might have specific APIs access. How can we do this in Oqtane?

**Comment by sbwalker** [comment, marked as answer]
Swagger has a ton of configuration options... so I am sure it can support your needs. You can create a custom swagger.json file at "/swagger/3.0.1/swagger.json" and add your options to modify the behavior.

**Reply by Sharafudeen** [reply]
Could you please give me an example or article links to follow your suggestions?

**Comment by hishamco** [comment]
@Sharafudeen did you treat your departments as tenants? If Yes you may config Swagger per tenant

**Reply by Sharafudeen** [reply]
I will try also this option but currently I didn't meet this requirements.

Thanks for suggesting..
